content/en/docs/getting-started/setup/create-cluster/gke.md

```diff
@@ -0,0 +1,124 @@
+| version\_stream\_ref | The git ref for version stream to use when booting Jenkins X. Refer to [version stream](/docs/concepts/version-stream) for more information. | `string` | `"master"` | no |
+| version\_stream\_url | The URL for the version stream to use when booting Jenkins X. Refer to [version stream](/docs/concepts/version-stream) for more information. | `string` | `"https://github.com/jenkins-x/jenkins-x-versions.git"` | no |
+| webhook | Jenkins X webhook handler for git provider | `string` | `"prow"` | no |
+| zone | Zone in which to create the cluster | `string` | `"us-central1-a"` | no |
+
+### Outputs
+
+| Name | Description |
+|------|-------------|
+| backup\_bucket\_url | The URL to the bucket for backup storage |
+| cluster\_name | The name of the created K8s cluster |
+| gcp\_project | The GCP project in which the resources got created in |
+| log\_storage\_url | The URL to the bucket for log storage |
+| report\_storage\_url | The URL to the bucket for report storage |
+| repository\_storage\_url | The URL to the bucket for artefact storage |
+| vault\_bucket\_url | The URL to the bucket for secret storage |
+| zone | The zone of the created K8s cluster |
+
+## Using a custom domain
+
+If you want to use a custom domain with your Jenkins X installation, you need to provide values for the variables _parent_domain_ and _tls_email_.
+_parent_domain_ is the fully qualified domain name you want to use and _tls_email_ is the email you want to use for issuing Let's Encrypt TLS certificates.
+
+Before you run the Terraform configuration, you also need to create a [Cloud DNS managed zone](https://cloud.google.com/dns/zones), with the DNS name in the managed zone matching your custom domain name, for example in the case of _example.jenkins-x.rocks_ as domain:
+
+![Creating a Managed Zone](/images/getting-started/create_managed_zone.png)
+
+When creating the managed zone, a set of DNS servers get created which you need to specify in the DNS settings of your DNS registrar.
+
+![DNS settings of a Managed Zone](/images/getting-started/managed_zone_details.png)
+
+It is essential that before you run `jx boot`, your DNS servers settings are propagated, and your domain resolves.
+You can use [DNS checker](https://dnschecker.org/) to check whether your domain settings have propagated.
+
+When a custom domain is provided, Jenkins X uses [ExternalDNS](https://github.com/kubernetes-sigs/external-dns) together with [cert-manager](https://github.com/jetstack/cert-manager) to create A record entries in your managed zone for the various exposed applications.
+
+If _parent_domain_ id not set, your cluster will use [nip.io](https://nip.io/) in order to create publicly resolvable URLs of the form ht<span>tp://\<app-name\>-\<environment-name\>.\<cluster-ip\>.nip.io.
```

Comment:
```suggestion
If _parent_domain_ is not set, your cluster will use [nip.io](https://nip.io/) in order to create publicly resolvable URLs of the form http://\<app-name\>-\<environment-name\>.\<cluster-ip\>.nip.io.
```

Reply:
The span is actually intentional to prevent the URL being rendered as a clickable URL. This is just meant as an example URL schema. On Markdown on GitHub using the span is the easiest way to prevent this. Not sure whether Hugo has something else.

Reply:
you should be able to add back-ticks around the url to avoid Hugo trying to make it clickable. 

I recall that there's another way to tell Hugo not to render it as a URL, but i can't seem to find it now; back-ticks might be your best bet